driver_integration_test.go

```diff
@@ -123,3 +127,32 @@ func TestDriverOpenDefaultEngine(t *testing.T) {
 func TestDriverExecStatement(t *testing.T) {
 	runTestDriverExecStatement(t, dsnMock)
 }
+
+// TestDriverSystemEngine checks system engine queries are executed without error
+func TestDriverSystemEngine(t *testing.T) {
+	suffix := strings.ReplaceAll(uuid.New().String(), "-", "")
+	databaseName := fmt.Sprintf("gosdk_system_engine_test_%s", suffix)
+	engineName := fmt.Sprintf("gosdk_system_engine_test_e_%s", suffix)
+	engineNewName := fmt.Sprintf("gosdk_system_engine_test_e_2_%s", suffix)
+
+	db, err := sql.Open("firebolt", dsnSystemEngineMock)
+	if err != nil {
+		t.Errorf("failed unexpectedly with %v", err)
+	}
+	queries := []string{
+		fmt.Sprintf("CREATE DATABASE %s", databaseName),
+		fmt.Sprintf("CREATE ENGINE %s WITH SPEC = 'C1' SCALE = 1", engineName),
+		fmt.Sprintf("ATTACH ENGINE %s TO %s", engineName, databaseName),
```

Comment:
Should we also test DETACH ENGINE? 
Also, SHOW [DATABASES|ENGINES]

Reply:
DETACH ENGINE is [being deprecated ](https://docs.firebolt.io/sql-reference/commands/detach-engine.html).
I just added SHOW DATABASES/SHOW ENGINES to the list